client/my-sites/plans-v2/utils.ts

```diff
@@ -215,13 +227,91 @@ export function itemToSelectorProduct(
 			description: item.getDescription(),
 			monthlyProductSlug,
 			term: item.term === TERM_BIENNIALLY ? TERM_ANNUALLY : item.term,
-			features: { items: [] },
+			features: {
+				items: buildCardFeaturesFromItem( item ),
+				more: {
+					url: PLAN_COMPARISON_PAGE,
+					label: translate( 'See all features' ),
+				},
```

Comment:
Just out of curiosity, by adding the `more` property to all plans, aren't we displaying the `See all features` link inside every Plan card?

Reply:
Exactly. It seems it'll be the case given the copydeck.